src/utils/formik-helpers.ts

```diff
@@ -75,6 +76,18 @@ export const getTabErrors = (
   return tabErrors;
 };
 
+export const getProjectTabErrors = (errors: FormikErrors<CristinProject>, touched?: FormikTouched<CristinProject>) => {
```

Comment:
Allerede ganske stor denne fila, så det kan være at argument for å flytte alt prosjekt-spesifikt til `project-form-helpers` elns. Kanskje samme med registration også, sånn at det til slutt er:
- commonFormHelpers.ts
- registrationFormHelpers.ts
- projectFormHelpers.ts

Reply:
God idé! Jeg tror ikke jeg bør gjøre alt det i denne PR'en, for det medfører at vi må teste en del også i registreringsskjemaet. Men jeg henter ut alle som er for project, og lager en ny folder-struktur som klargjør for det.